Makefile

```diff
@@ -110,9 +110,37 @@ deploy-full-local-setup: ## Deploy full local multicluster setup
 	$(call deploy-local-cluster,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),$(VERSION),)
 	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(VERSION),$(CLUSTER_GSLB2_HELM_ARGS))
 
+.PHONY: deploy-stable
+deploy-stable:
+	@echo "\n$(YELLOW) import $(CYAN)k8gb:$(VERSION) $(YELLOW)to $(CYAN)$(CLUSTER_GSLB1), $(CLUSTER_GSLB2) $(NC)"
+	$(call deploy-local-cluster,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),$(VERSION),)
+	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(VERSION),$(CLUSTER_GSLB2_HELM_ARGS))
+
+	$(call list-running-pods,$(CLUSTER_GSLB1))
+	$(call list-running-pods,$(CLUSTER_GSLB2))
+
+.PHONY: deploy-candidate-with-helm
+deploy-candidate-with-helm:
+	@echo "\n$(YELLOW)build k8gb docker and import to $(CYAN)$(CLUSTER_GSLB1), $(CLUSTER_GSLB2) $(NC)"
+	docker build . -t $(REPO):$(SEMVER)
+
+	k3d image import $(REPO):$(SEMVER) -c $(CLUSTER_GSLB1)
+	k3d image import $(REPO):$(SEMVER) -c $(CLUSTER_GSLB2)
+
+	@echo "\n$(YELLOW)Upgrade GSLB operator from $(VERSION) to $(SEMVER) on k3d-$(CLUSTER_GSLB1) $(NC)"
+	kubectl config use-context k3d-$(CLUSTER_GSLB1)
+	$(call deploy-k8gb-with-helm,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),$(SEMVER),)
+
+	@echo "\n$(YELLOW)Upgrade GSLB operator from $(VERSION) to $(SEMVER) on k3d-$(CLUSTER_GSLB2) $(NC)"
+	kubectl config use-context k3d-$(CLUSTER_GSLB2)
+	$(call deploy-k8gb-with-helm,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(SEMVER),$(CLUSTER_GSLB2_HELM_ARGS))
+
+	$(call list-running-pods,$(CLUSTER_GSLB1))
+	$(call list-running-pods,$(CLUSTER_GSLB2))
+
 # triggered by terraform GitHub Action. Clusters already exists. GO is not installed yet
-.PHONY: deploy-to-AbsaOSS-k3d-action
-deploy-to-AbsaOSS-k3d-action:
+.PHONY: deploy-candidate
+deploy-candidate:
 	@echo "\n$(YELLOW)build k8gb docker and push to registry $(NC)"
```

Comment:
This stuff still tells 'push to registry' can we make it consistent with https://github.com/AbsaOSS/k8gb/pull/429/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R124 ?

Reply:
sorry, see #435